Index: test.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>print('git test')\r\nprint('wenhe testing')
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test.py b/test.py
--- a/test.py	(revision 653f3da21ab13583d0c71f3b89856416acd19ebd)
+++ b/test.py	(date 1658721080452)
@@ -1,2 +1,2 @@
 print('git test')
-print('wenhe testing')
\ No newline at end of file
+print('wenhe  testing')
\ No newline at end of file
